ingestion/src/metadata/clients/domo_client.py

```diff
@@ -74,14 +76,24 @@ def get_chart_details(self, page_id) -> dict:
         return None
 
     def get_pipelines(self):
-        response = self.client._request(  # pylint: disable=protected-access
-            method="GET", path=WORKFLOW_URL, headers=HEADERS
-        )
-        return response
+        try:
+            response = self.client._request(  # pylint: disable=protected-access
+                method="GET", path=WORKFLOW_URL, headers=HEADERS
+            )
+            return response
+        except Exception as exc:
+            logger.info(f"Error while getting pipelines - {exc}")
+            logger.debug(traceback.format_exc())
+        return []
 
     def get_runs(self, workflow_id):
-        url = f"dataprocessing/v1/dataflows/{workflow_id}/executions?limit=100&offset=0"
-        response = self.client._request(  # pylint: disable=protected-access
-            method="GET", path=url, headers=HEADERS
-        )
-        return response
+        try:
+            url = f"dataprocessing/v1/dataflows/{workflow_id}/executions?limit=100&offset=0"
```

Comment:
Here we are only ingesting 100 executions, ideally we should fetch all the runs right. 🤔 ?

Reply:
I will open a different pr for this